### DISCUSSION TITLE
The commit record is changed automatically

### DISCUSSION THREAD

**Discussion by zhaijunxiao** [community/community#56924] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

HI.I'm having a very strange problem. 
I invited two people to develop my project together. 
When I removed one of them, something magical happened.
All of his committers were turned into someone else I had never invited to my project.

**Comment by Gustavonn07** [comment]
When you remove someone from a project, it's unusual for their contributions to be automatically attributed to someone else who was not originally involved. There could be reasons why this occurred:

1. Miscommunication or misunderstanding: It's possible that there was a miscommunication or misunderstanding regarding the project roles or permissions. Double-check the settings and permissions in your project management platform to ensure that the appropriate people have the correct access.

2. Technical glitch or bug: There could have been a technical glitch or bug in the project management software or version control system you are using. Such issues can sometimes result in unexpected behavior. Consider reaching out to the support team of the Github to report the problem and seek assistance.

3. Unauthorized access or malicious activity: It's unlikely but possible that someone gained unauthorized access to your project or the accounts involved. This could lead to changes in contributors or other unexpected behaviors. In such cases, it's crucial to review the security measures of your project and take appropriate action, such as changing passwords and enabling two-factor authentication.

To resolve the issue, I recommend taking the following steps:

1. Review project logs: If your project management platform provides logs or an audit trail, review them to see if there is any record of the contributor changes. This might help you identify any unexpected actions or patterns.

2. Communicate with the individuals involved: Reach out to the people who were originally invited and the person whose committers were mistakenly changed. Discuss the situation with them to understand if they have any insights or if they made any unintended changes.

3. Contact platform support: If the issue persists or you suspect a technical glitch, contact the support team of the project management platform you're using. Provide them with the details of the problem and any relevant information they might need to investigate the issue further.

**Comment by airtower-luna** [comment]
> All of his committers were turned into someone else I had never invited to my project.

Do you mean *commits*? Those should never change unless you explicitly edit the repository history, that's one of the core concepts of Git. If you have a local clone you can easily compare the commit IDs to verify they haven't changed.

I'm guessing here, but if the repository is private I wouldn't be surprised if instead of the username the commits are now shown with the author name recorded in the commit. :thinking:
